examples/data_science/tensorflow/devbox.json

```diff
@@ -1,8 +1,8 @@
 {
   "packages": [
-    "python310",
     "python310Packages.pip",
-    "stdenv.cc.cc.lib"
+    "stdenv.cc.cc.lib",
+    "python310@latest"
```

Comment:
we should make a pass (not blocking) to update all such examples to be of the modern form: `python@3.10`

Reply:
Agreed. This pr was mostly automated stuff (which makes it large and hard to review) so I was hoping to keep manual changes to minimum. I can do that in follow up.